services/github/github-actions.tester.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { isBuildStatus } = require('../build-status')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('nonexistent repo')
+  .get('/badges/shields-fakeness/fake.json')
+  .expectBadge({
+    label: 'build',
+    message: 'repo, branch, or workflow not found',
+  })
+
+t.create('nonexistent workflow')
+  .get('/actions/toolkit/not-a-real-workflow.json')
+  .expectBadge({
+    label: 'build',
+    message: 'repo, branch, or workflow not found',
+  })
+
+t.create('valid workflow')
+  .get('/actions/toolkit/Main%20workflow.json')
+  .expectBadge({
+    label: 'build',
+    message: isBuildStatus,
```

Comment:
Maybe worth adding the `no status` case to avoid flaky tests?

Reply:
Good catch!